subsys/bootloader/debug/Kconfig

```diff
@@ -66,4 +66,10 @@ config SB_SEGGER_RTT_MEMCPY_USE_BYTELOOP
 	bool "Use a simple byte-loop instead of standard memcpy"
 endmenu
 endif # SB_DEBUG_PORT_SEGGER_RTT
+
+config SB_BPROT_IN_DEBUG
```

Comment:
Check for HAS_HW_NRF_MPU as well.

You could do

    if (HAS_HW_NRF_BPROT || HAS_HW_NRF_MPU)

instead of depends on, so it will be invisible except when needed.

Reply:
you know `if foo` and `depends on foo` is equivalent? so it's invisible if `HAS_HW_NRF_BPROT` is not set.

I'll add the `HAS_HW_NRF_MPU` this symbol is only set by nrf51 boards and not boards which has ARM MPU right? And the nrfx hal layer abstracts it away to match the BPROT API?

Reply:
Correct, it has nothing to do with ARM MPU. The abstractions to match BPROT are partly implemented in fprotect_bprot.c as well.

Reply:
Added the suggestion for matching against `NRF_MPU`also now :) so I'll mark this as resolved as I would like to keep the depends on instead of wrapping it in a `if` as it's much cleaner.